<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moroccan Card Game </string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>401</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="computer">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="computer1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="computer2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="computer3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="computer4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>401</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="player">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="ClickableLabel" name="player1">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ClickableLabel" name="player2">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ClickableLabel" name="player3">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ClickableLabel" name="player4">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/blank.gif</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="deck">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>210</y>
     <width>121</width>
     <height>191</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/back.gif);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>121</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/back.gif</pixmap>
    </property>
   </widget>
   <widget class="ClickableLabel" name="fillLabel">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/back.gif</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="playArea1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea9_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea10">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea11">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="computerScoreLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Score: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Score: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="playArea15">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea14">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea17">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea13">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playArea16">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>95</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/blank.gif</pixmap>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
